src/ForwardDiffNumber.jl

```diff
@@ -128,6 +131,9 @@ isfinite(n::ForwardDiffNumber) = isfinite(value(n))
 isinf(n::ForwardDiffNumber) = isinf(value(n))
 isreal(n::ForwardDiffNumber) = isconstant(n)
 
+floor{T}(::Type{T}, n::ForwardDiffNumber) = floor(T, value(n))
```

Comment:
I'm suggesting it should return a ``ForwardDiffNumber`` with floor applied to the realpart, like the implementation of ``mod`` [here](https://github.com/JuliaDiff/DualNumbers.jl/blob/7d4eedfa1bd6c0af7d844d085d0c2ce6b2357aa7/src/dual.jl#L238).